addons/point_of_sale/static/tests/tours/PaymentScreen.tour.js

```diff
@@ -188,4 +188,21 @@ PaymentScreen.check.changeIs('3.05');
 PaymentScreen.check.totalDueIs('1.95');
 Chrome.do.clickTicketButton();
 
-Tour.register('PaymentScreenTotalDueWithOverPayment', { test: true, url: '/pos/ui' }, getSteps());
+    startSteps();
+
+    ProductScreen.do.confirmOpeningPopup();
+    ProductScreen.do.clickHomeCategory();
+    ProductScreen.exec.addOrderline('Product Test', '1');
+    ProductScreen.do.clickPayButton();
+
+    PaymentScreen.check.totalIs('1.95');
+    PaymentScreen.do.clickPaymentMethod('Cash');
+    PaymentScreen.do.pressNumpad('5');
+
+    PaymentScreen.check.remainingIs('0.0');
+    PaymentScreen.check.changeIs('3.05');
+    PaymentScreen.check.totalDueIs('1.95');
+
+    Tour.register('PaymentScreenTotalDueWithOverPayment', { test: true, url: '/pos/ui' }, getSteps());
```

Comment:
Find the code that you changed in the previous version, and do the same replacement here. In this case, the code you need to change is in line 184.